RevenueCatUI/CustomerCenter/ViewModels/ManageSubscriptionsViewModel.swift

```diff
@@ -118,20 +144,42 @@ class ManageSubscriptionsViewModel: ObservableObject {
         )
     }
 
-    #if os(iOS) || targetEnvironment(macCatalyst)
+#if os(iOS) || targetEnvironment(macCatalyst)
```

Comment:
xcode keeps formatting this like this, so I have up and I think this might be the way to go?